docs/book/src/reference/owner_references.md

```diff
@@ -0,0 +1,79 @@
+# Owner References
+
+
+Cluster API uses [Kubernetes owner references](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/) to track relationships between objects. These references are used for Kubernetes garbage collection, which is the basis of Cluster deletion in CAPI. They are also used places where the ownership hierarchy is important, for example when using `clusterctl move`.
+
+CAPI uses owner references in an opinionated way. The following guidelines should be considered:
+1. Objects should always be created with an owner reference to prevent leaking objects. Initial ownerReferences can be replaced later where another object is a more appropriate owner.
+2. Owner references should be re-reconciled if they are lost for an object. This is required as some tools - e.g. velero - may delete owner references on objects.
+3. Owner references should be kept to the most recent apiVersion.
+   - This ensures garbage collection still works after an old apiVersion is no longer served.
+4. Owner references should not be added unless required.
+   - Multiple owner references on a single object should be exceptional.
+
+
+  
+
+## Owner reference relationships in Cluster API
+
+The below tables map out the a reference for ownership relationships for the objects in a Cluster API cluster.
+Note: Providers may implement their own ownership relationships which may or may not map directly to the below tables.
+
+## Kubernetes core types
+
+| type      | Owner               | Note                     |
+|-----------|---------------------|--------------------------|
+| Secret    | KubeadmControlPlane | For cluster certificates |
+| Secret    | KubeadmConfig       | For bootstrap secrets    |
+| Secret    | ClusterResourceSet  | When created by CRS      |
+| ConfigMap | ClusterResourceSet  |                          |
```

Comment:
Also when created by CRS? (and I assume not covered by our e2e verifications because we don't have this case in our e2e tests?)

Reply:
We have the configMap case but don't have the secret case in our e2e test AFAIK

Reply:
Yup that's fine. was just curious